### ISSUE TITLE
Chemotion 1.5.2 with NMRium-Wrapper 0.2.1 ends in white-screen of death

### ISSUE THREAD

**Issue by ptrxyz** [ComPlat/chemotion_ELN#1276]
When using Chemotion 1.5.2 together with Spectra 0.11.1 and NMRium 0.2.1 I get a white screen of death. This is what I do:

Create a new sample, add an analysis, add a JCAMP file, set onthology to 1H. Save the sample.
Open NMRium using the button in the sample's details panel.

See the video:
[Peek 2023-04-15 04-12.webm](https://user-images.githubusercontent.com/209056/232178239-0fd7ff27-a259-4df9-a7ca-4679e58b6ab6.webm)

The JCAMP file I was using:
[SCR350.jcamp.zip](https://github.com/ComPlat/chemotion_ELN/files/11238084/SCR350.jcamp.zip)

The spectra config:
```yml
production:
  :chemspectra:
    :url: 'https://011.spectra.hydrogen.scc.kit.edu'
  :nmriumwrapper:
    :url: 'https://nmrium-0-2-1-secure.eln.hydrogen.scc.kit.edu/'
```

**Comment by baolanlequang** [comment]
This issue relate with https://github.com/ComPlat/chemotion_ELN/issues/1274

**Comment by PiTrem** [comment]
resolved with https://github.com/ComPlat/chemotion_ELN/pull/1275 

shipped in 1.5.3
